client/src/components/Header/components/universal-nav.css

```diff
@@ -176,7 +176,7 @@ button.nav-link:focus {
 button.nav-link:focus:not(:focus-visible) {
   background-color: inherit;
   color: #fff;
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert this back to `outline: none`. This rule set is for when the button has focus but it was caused by something other than the keyboard (e.g. mouse, touch). In that case we don't want a focus outline to show.